### ISSUE TITLE
Default loss function

### ISSUE THREAD

**Issue by mrybak-Novetta** [rosinality/style-based-gan-pytorch#16]
I noticed that the loss function is set to wp_gan by default. however, in the official implementation, the loss is set to r1 by default. Is that correct? If so why did you choose that?

**Comment by rosinality** [comment]
In the paper authors said they used wgan-gp for celeba-hq. I have tested my implementation using celeba first so I have used wgan-gp. Later I added non saturating loss and r1 for ffhq.
